packages/rehype-mathjax/renderer.js

```diff
@@ -0,0 +1,25 @@
+const {mathjax} = require('mathjax-full/js/mathjax')
+const {TeX} = require('mathjax-full/js/input/tex')
+const {SVG} = require('mathjax-full/js/output/svg')
+const {liteAdaptor} = require('mathjax-full/js/adaptors/liteAdaptor')
+const {
+  RegisterHTMLHandler: registerHTMLHandler
+} = require('mathjax-full/js/handlers/html')
+const {AllPackages} = require('mathjax-full/js/input/tex/AllPackages')
+const adaptor = liteAdaptor()
+registerHTMLHandler(adaptor)
+const tex = new TeX({packages: AllPackages})
+const svg = new SVG({fontCache: 'none'})
+const html = mathjax.document('', {InputJax: tex, OutputJax: svg})
+
+module.exports = (math, display) => {
+  const css = adaptor.textContent(svg.styleSheet(html))
+  const pic = adaptor.outerHTML(html.convert(math, {display}))
```

Comment:
Instead of serializing the mathjax stylesheet and html nodes, could we map them to hast?
hast are the nodes we use, and because katex doesn’t have adapters we need to use parse5 (we opened an issue with katex), but as mathjax does support adapters, we should be able to shave off 50kb minzipped by not using p5.